tsconfig.json

```diff
@@ -39,5 +39,7 @@
   "exclude": [
     "node_modules",
     "dist"
-  ]
+  ],
+  "vueCompilerOptions": {
+  }
```

Comment:
This is to solve this "known issue":

![image](https://user-images.githubusercontent.com/10002889/194602803-9b4af42c-fffd-4265-afa7-deeb96bc0a55.png)